packages/app/pages/course/[cid]/insights.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { ReactElement } from "react";
+import { API } from "@koh/api-client";
+import useSWR from "swr";
+import { Tooltip, Card } from "antd";
+import { InfoCircleOutlined } from "@ant-design/icons";
+import { useRouter } from "next/router";
+import { StandardPageContainer } from "../../../components/common/PageContainer";
+import SimpleDisplayComponent from "../../../components/Insights/components/SimpleDisplayComponent";
+import SimpleChartComponent from "../../../components/Insights/components/SimpleChartComponent";
+
+export default function Insights(): ReactElement {
+  // TODO: In the future this will come from the users specific insights that want to see
+  const insights = ["totalQuestionsAsked", "totalStudents"];
+
+  return (
+    <>
+      <StandardPageContainer>
+        <h1 style={{ margin: "20px" }}>Insights Dashboard</h1>
+        <divider />
+        <div style={{ display: "flex", direction: "ltr" }}>
+          {insights?.map((insightName: string) => {
+            return (
+              <RenderInsight key={insightName} insightName={insightName} />
+            );
+          })}
+        </div>
+      </StandardPageContainer>
+    </>
+  );
+}
+
+interface RenderInsightProps {
+  insightName: string;
+}
+
+function RenderInsight({ insightName }: RenderInsightProps): ReactElement {
+  const router = useRouter();
+  const { cid } = router.query;
+
+  const { data: insight, error, mutate } = useSWR(
+    cid && `api/v1/insights/${cid}/${insightName}`,
+    async () => API.insights.get(Number(cid), insightName)
```

Comment:
I'm *slightly* concerned about performance here -- since we're making a separate API request for each insight -- would it be better to batch them all up? @dajinchu would probably be the best person to ask (though, also, to be fair, this isn't a high-traffic endpoint, so we might be ok).

For once GraphQL would make sense here haha

Reply:
I think it's fine either way. Http overhead isn't that crazy. Having them separate might be better so they they can load in progressively, if one chart requires a ton of data it won't block the others from displaying

Reply:
We started with the endpoint retuning all the insights in one request but having the ability to load the page progressively we thought would be a better UX.

Reply:
Just for personal growth, why would GraphQL be beneficial here?